runtime/common/Cargo.toml

```diff
@@ -28,6 +28,7 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 
 # Polkadot dependencies
 xcm = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.38" }
+xcm-executor = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.38" }
```

Comment:
Please add `xcm-executor/std` and `xcm-executor/runtime` features for to the crate.